server/src/model/chapters.go

```diff
@@ -307,3 +308,15 @@ func GetAllChaptersByRegion(db *sqlx.DB) (map[string][]ChapterWithToken, error)
 	//return pages grouped into regions, nil
 	return regions, nil
 }
+
+const SFBayPageID = 1377014279263790
+const NorthBayPageID = 61569960364321
+
```

Comment:
nit: should we put these consts somewhere along w/ the other page id consts?

Reply:
done! moved the ALC one here